components/engine_rocks/src/properties.rs

```diff
@@ -408,7 +408,10 @@ impl TablePropertiesCollector for MvccPropertiesCollector {
         // TsFilter filters sst based on max_ts and min_ts during iterating.
         // To prevent seeing outdated (GC) records, we should consider
         // RocksDB delete entry type.
-        if entry_type != DBEntryType::Put && entry_type != DBEntryType::Delete {
+        if entry_type != DBEntryType::Put
```

Comment:
components/engine_rocks/src/ttl_properties.rs
Does this file need similar change?